hack/charts/2.0.0/images.json

```diff
@@ -0,0 +1,27 @@
+{
+   "pachd": {
+      "repository": "pachyderm/pachd",
+      "tag": "latest",
+      "pullPolicy": "IfNotPresent"
+   },
+   "dash": {
+      "repository": "pachyderm/haberdashery",
+      "tag": "b6769ac1ad4561d76e9382bc24ac271bc7686956",
+      "pullPolicy": "IfNotPresent"
+   },
+   "etcd": {
+      "repository": "pachyderm/etcd",
```

Comment:
We're not currently seeking container approval for this—does Red Hat know of an approved etcd image we can use?